scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -338,21 +339,25 @@ task AnnotateVCF {
     String path_reference = "/References/Homo_sapiens.GRCh38.Nirvana.dat"
 
     command <<<
-        # set -e
-
         bash ~{monitoring_script} > monitoring.log &
 
         # Prepend date, time and pwd to xtrace log entries.
         PS4='\D{+%F %T} \w $ '
         set -o errexit -o nounset -o pipefail -o xtrace
 
-        # =======================================
-        # Handle our data sources:
+        # There's an issue with how the projects/broad-dsde-cromwell-dev/global/images/nirvana-3-18-1-references-2023-01-03
+        # disk image was built: while all the reference files do exist on the image they are not at the expected
+        # locations. The following code works around this issue and should continue to work even after a corrected
+        # version of the Nirvana reference image is deployed into Terra.
+
+        # Find where the reference disk has been mounted on this VM:
+        REFERENCE_DISK_MOUNT_POINT=$(lsblk | sed -E -n 's!.*(/mnt/[a-f0-9]+).*!\1!p')
```

Comment:
see that this fails clearly if reference volume is not found (like 'Use reference disks' was not checked)

Reply:
confirmed! https://job-manager.dsde-prod.broadinstitute.org/jobs/758ae840-21bb-4d52-ad34-7d62a3fb81aa